packages/vscode/CONTRIBUTING.md

```diff
@@ -41,6 +41,10 @@ On push to the master branch, a new Insider extension is released, with an incre
 - Make a change to the syntax
 - To reload, press the reload button in VSCode ( **Developer: Inspect TM Scopes** is helpful for debugging syntax issues )
 
+### Dependencies
+
+- the version of `@types/vscode` must always be smaller or equal to the `engines.vscode` version. Otherwise the extension can not be packaged.
```

Comment:
Will this fail some tests somehow?

Reply:
no, the build step would fail in this case and a slack notification would be sent then.

Reply:
But maybe it'd be good to have a test for that yes.

Reply:
It would ensure that we can not even merge and PRs that come in via Renovate.